old-ui/app/components/send/send-token.js

```diff
@@ -63,7 +63,7 @@ class SendTransactionScreen extends PersistentForm {
           <EnsInput
             name="address"
             placeholder="Recipient Address"
-            onChange={() => this.recipientDidChange.bind(this)}
+            onChange={this.recipientDidChange.bind(this)}
```

Comment:
While I was testing the RNS integration, I found a bug with ENS integration when sending tokens. This line fixes it, and it is included in the current PR because it is strongly related to the main purpose of this PR.

Reply:
Ah, so it wasn't working before, even for ENS - got it!

Reply:
@javiesses Nice catch!